src/proto/streams/prioritize.rs

```diff
@@ -791,7 +791,10 @@ impl Prioritize {
                         }),
                         None => {
                             if let Some(reason) = stream.state.get_scheduled_reset() {
-                                stream.state.set_reset(reason);
+                                let stream_id = stream.id;
```

Comment:
Isn't this redundant? The stream already knows it's ID?

Reply:
Do you think it is important to remove that `stream_id` from there?

I think I could, if I inline `proto::Error` variants into `streams::state::Cause`, but honestly I don't think it is worth it.

Reply:
I don't think it's a big deal. I was just noting it, in case there was a better way that popped into your mind.